src/components/graphs/lineGraph.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Chart as ChartJS,
+  LinearScale,
+  CategoryScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  Filler
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  Filler
+);
+
+const LineGraph: React.FC<{
+  width: number,
+  height: number
+  yCallback: (value: any, index: any, ticks: any) => string,
+  graphLabel: string,
+  labels: string[],
+  dataPts: number[]
+}> = ({ dataPts, graphLabel, labels, yCallback, width, height }) => {
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        labels: {
+          font: {
+            size: 14,
+            family: "system-ui"
+          },
+          color: "#64748b"
```

Comment:
Add `color: "#FFFFFF"` to make title more visible

Reply:
Will not render properly in light mode. As of now, this is a compromise as the dynamic props do not re-render the graph based off changes in the page.